README.md

```diff
@@ -329,36 +216,40 @@ In this example we search for all elements with the `fsType` equals `Example`. W
 
 ```typescript
 fsxaApi.fetchByFilter({
-    filters: [
-      {
-        field: 'fsType',
-        operator: ComparisonQueryOperatorEnum.EQUALS,
-        value: 'Example',
-      },
-    ],
-    locale: 'en',
-    page: 2,
-    pagesize: 50,
-    additionalParams: { keys: { identifier: 0 } },
-    sort: [{name:'fsType', order:'desc'}],
-  }
-)
+  filters: [
+    {
+      field: 'fsType',
+      operator: ComparisonQueryOperatorEnum.EQUALS,
+      value: 'Example',
+    },
+  ],
+  locale: 'en',
+  page: 2,
+  pagesize: 50,
+  additionalParams: { keys: { identifier: 0 } },
+  sort: [{ name: 'fsType', order: 'desc' }],
+})
 ```
 
 The default sorting is by the id descending. Multisort is possible and the first sort param is prioritized over subsequent. The sorting is happening on the raw data.
 
+> **_Attention_**<br>
+> The fieldnames and keys that can be passed to the `fetchByFilter` method have to match the key names that are present in the CaaS data.
+
 ### fetchProjectProperties
 
 Returns the project properties of the given language.
 
-Expects as input parameter the language abbreviation.
+Expects an object parameter with the language abbreviation property
```

Comment:
Expects a parameter object with the locale.

Reply:
I will take that recommendation